src/widgetastic_patternfly/__init__.py

```diff
@@ -1380,3 +1380,58 @@ def __init__(self, parent, id=None, name=None, logger=None):
     @property
     def selected(self):
         return self.browser.is_selected(parent=self, locator=self.input)
+
+
+class AboutModal(Widget):
+    """
+    Represents the patternfly about modal
+
+    Provides a close method
+    """
+    ROOT = ParametrizedLocator('//div[normalize-space(@id)={@id|quote} and '
+                               'contains(@class, "modal") and contains(@class, "fade")]')
+    CLOSE_LOC = './/div[@class="modal-header"]/button[@class="close" and @data-dismiss="modal"]'
+    TITLE_LOC = './/div[@class="modal-body"]/h2'
```

Comment:
I believe if you define this as a Text widget, it should work as well. Similar with some of the other locators.